### PR TITLE
march 31

### PR DESCRIPTION
**Pull request by anhie92**
removed some unneeded code, moved search to nav bar, added favorites in user page

### CODE REVIEW

File: src/app/media/media.component.ts [hunk 1]
```diff
@@ -71,7 +71,7 @@ export class MediaComponent {
       });
       this.selectedMedia=true;
     }
-    if(mediaType="show")
+    else if(mediaType="show")
```
Comment (SuperBus20) [inline src/app/media/media.component.ts]: This line and line 66 need changed to be a double or tripple =. (mediaType=="show") and (mediaType=="movie") respectively. As is it its setting mediaType to show/media not checking to see if mediaType is actually a show or movie

**Review by SuperBus20 — DISMISSED** [review]
Line 66 and 74 need changed in the media component.

File: src/app/media/media.component.ts (hunk 1 above)
Comment (anhie92) [reply src/app/media/media.component.ts]: Will change it soon

**Review by anhie92 — COMMENTED** [review]
changed junes comment

**Review by Nmlycette — APPROVED** [review]

**Review by Nmlycette — APPROVED** [review]

**Review by Nmlycette — APPROVED** [review]

**Pull request merged**